03_algorithm_selection.ipynb

```diff
@@ -17,7 +17,7 @@
    "source": [
```

Comment:
<p>It might be nice to mention that while the classical definition for a perceptron is around activations, the term is used ambiguously in text e.g. a multi-layer perceptron can simply mean a feedforward network where the outputs of a node are the weights and biases without thresholding or activations (leading to an output range in the set of real numbers). The expansion of this explanation could also allow for a description of two common ML problems: exploding and vanishing gradient. Where output restrictions like activations reduce exploding gradient issues and typically architecture decisions remedy vanishing gradients (RNN variants, or Skip/residual connections come to mind) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/ml_weather_tutorial/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>